ci/Jenkinsfile.tests

```diff
@@ -29,6 +29,11 @@ pipeline {
       description: 'Specify build type. Values: pr / e2e / nightly / release',
       defaultValue: 'pr',
     )
+    booleanParam(
+      name: 'RUN_CONTRACT_TESTS',
+      description: 'Whether to run optional and slow contract tests.',
```

Comment:
are they slow or just timing out? 🤔

Reply:
Those options are not mutually exclusive. Also, no idea.